react/src/components/ImageList.tsx

```diff
@@ -330,7 +424,27 @@ const ImageList: React.FC<{ style?: React.CSSProperties }> = ({ style }) => {
           ...style,
         }}
       >
-        <Flex justify="end" style={{ padding: token.paddingSM }}>
+        <Flex justify="end" style={{ padding: token.paddingSM, gap: 8 }}>
+          <Input
+            allowClear
+            prefix={<SearchOutlined />}
+            placeholder={t('environment.SearchImages')}
+            onChange={(e) => {
+              startTransition(() => setImageSearch(e.target.value));
```

Comment:
I didn't use loading when updating information by search or control. I thought the updates were fast enough that I didn't need to use loading, and I found the UI with loading to be more cluttered.

Reply:
Please add a loading indicator to the table when searching and updating the table. It's better if there is a possibility of the UI displaying inconsistent data.